nexus/tests/integration_tests/switch_port.rs

```diff
@@ -118,7 +118,15 @@ async fn test_port_settings_basic_crud(ctx: &ControlPlaneTestContext) {
         "phy0".into(),
         LinkConfigCreate {
             mtu: 4700,
-            lldp: LldpServiceConfigCreate { enabled: false, lldp_config: None },
+            lldp: LldpLinkConfigCreate {
+                enabled: true,
+                link_name: Some("Link Name".into()),
+                link_description: Some("link_ Dscription".into()),
```

Comment:
super nit, because this is a test-only string and really doesn't matter, but:

```suggestion
                link_description: Some("link description".into()),
```